src/content/docs/bff/extensibility/management/silent-login.md

```diff
@@ -12,15 +12,15 @@ redirect_from:
   - /identityserver/v7/bff/extensibility/management/silent-login/
 ---
 
-The BFF silent login endpoint can be customized by implementing the *ISilentLoginService* or by extending *DefaultSilentLoginService*, its default implementation.
+The BFF silent login endpoint can be customized by implementing the *ISilentLoginEndpoint*.
```

Comment:
So I implemented the interface, but I'm not really sure how to register and override the current implementation. There isn't much code here for that.